src/mod2c_core/simultan.c

```diff
@@ -243,10 +243,6 @@ numlist, numlist-1, counts);
 	  , numlist-1, counts, numlist, SYM(q2)->name, suffix, numlist);
 	vectorize_substitute(qret, buf);
 	Insertstr(q3, "/*if(_reset) {abort_run(_reset);}*/ }\n");
-	Sprintf(buf,
-	  "extern int _newton_%s%s(_threadargsproto_);\n"
-	  , SYM(q2)->name, suffix);
-	Linsertstr(procfunc, buf);
```

Comment:
above declaration is actually not extern. For example, look at hippocampus branch (`sandbox/king/hippocampus`) and compile `cacumm.mod` file. You will see that `extern int _newton_integrate_cacum(_threadargsproto_);` is defined in the translated c file. So I think extern here is misleading (and we shouldn't remove it (?) ).